File: Conscious Python - OOP/step 03. Наследование/3.1/3.1.01.py
'''

    Создайте класс Gomer.
    Создайте в классе Gomer метод __init__(self, name), который создает атрибут name.
    Создайте пустой класс Daughter и сделайте его дочерним классом для Gomer. Используйте pass для пустого класса.
    Создайте экземпляр daughter1 для класса Daughter и присвойте атрибуту name значение Lisa.
    Выведите на экран значение атрибута name через экземпляр daughter1.

Sample Input:


Sample Output:

Lisa

'''

class Gomer:
    def __init__(self, name):
        self.name = name
        
class Daughter(Gomer):
    pass

daughter1 = Daughter('Lisa')
print(daughter1.name)

File: Diving into JavaScript: For Beginners/step03/3.01/3.1.01.js
/*

Напишите программу, которая проверяет, является ли введенное пользователем число положительным. 
Если число положительное, программа выводит сообщение "Положительное", если не положительное - не выводит ничего.

Sample Input 1:

1

Sample Output 1:

Положительное

Sample Input 2:

0

Sample Output 2:

Sample Input 3:

-5

Sample Output 3:

*/

let number = Number(prompt());

if (number > 0) {
    console.log('Положительное');
}
